paddle/fluid/operators/cinn/cinn_launch_context.h

```diff
@@ -155,6 +159,10 @@ class CinnLaunchContext {
   // the variable scope compiled from cinn
   const std::shared_ptr<CinnScope> cinn_scope_;
 
+  std::unique_ptr<framework::ProgramDesc> runtime_program_;
```

Comment:
```suggestion
  std::unique_ptr<framework::ProgramDesc> runtime_program_desc_;
```
与现有cinn编译生成的runtime_program区分开

Reply:
Done, tks